.idea/vcs.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
```

Comment:
Why is this removed?

Reply:
Shall we just remove this file altogether?

Reply:
Apparently the docs say we should have .idea in VCS, and since this is a file in .idea that describes VCS, I'm now for just keeping it in.

Reply:
Also, checking out this branch causes my IDE to no longer recognise the project.

Reply:
Still unsure what the values in this file should really be. The project made this change by itself and it seems simpler from my POV.

Reply:
Ah OK. Wonder which file is causing the issue.

Reply:
I think it's the removal of app.iml

Reply:
I'm gona reject this pull request until we decide what to add/remove from .gitignore. 

See under 'Directory-based format' here: https://www.jetbrains.com/help/idea/2017.1/about-projects.html